### DISCUSSION TITLE
ACLS implementation

### DISCUSSION THREAD

**Discussion by Simba75013** [Khopa/dcs_liberation#2197] [Q&A] [answered]
Hi there !
With the arrival of the ACLS on DCS OB, do I have to manually edit every mission to activate it or is there a plan to make it automatically on that is in the works ?

**Comment by RndName** [comment, marked as answer]
For now the ACLS (and link4) have to be enabled manually in the mission by just adding the two new tasks to the carrier. Future Versions will have the ACLS and Link4 enabled by default.
